django/contrib/gis/db/backends/mysql/operations.py

```diff
@@ -31,6 +31,15 @@ def select(self):
     def from_text(self):
         return self.geom_func_prefix + "GeomFromText"
 
+    @cached_property
+    def collect(self):
+        if not self.connection.mysql_is_mariadb and self.connection.mysql_version >= (
+            8,
+            0,
+            24,
+        ):
```

Comment:
Can we use `connection.features.supports_collect_aggr` instead?

Reply:
Yeah, I think we can. To be honest I found the whole approach to the aggregation stuff quite confusing.

It might be better to change it to work like the functions stuff works... 🤔

Reply:
Maybe, suggestions welcome.